bin/supply

```diff
@@ -5,19 +5,19 @@
 BUILD_DIR=$1
 
 # To store assets needed during build (currently unused).
-CACHE_DIR=$2
+# CACHE_DIR=$2
 
 # App dependencies are stored in $DEPS_DIR/$INDEX_DIR.
 DEPS_DIR=$3
 INDEX_DIR=$4
 
-BIN_DIR=$(cd $(dirname $0); pwd)
-BUILDPACK_DIR=$(dirname $BIN_DIR)
+BIN_DIR=$(cd $(dirname "$0"); pwd)
```

Comment:
I'm seeing a CodeClimate warning on GitHub, I think it wants nested double quotes for this and the next line:
```
BIN_DIR="$(cd $(dirname "$0"); pwd)"
BUILDPACK_DIR="$(dirname "${BIN_DIR}")"/buildpack
```